packages/labs/analyzer/src/lib/references.ts

```diff
@@ -310,16 +331,14 @@ export const getExportReferences = (
           decNameOrRef: getImportReference(
             specifier,
             moduleSpecifier,
-            el.name.getText(),
+            localName,
             analyzer
           ),
         });
       } else {
         // Get the declaration for this symbol, so we can determine if
         // it was declared locally or not
-        const symbol = analyzer.program
-          .getTypeChecker()
-          .getSymbolAtLocation(el.name);
+        const symbol = getSymbolForName(localName, localNameNode, analyzer);
```

Comment:
Given
```ts
import {Foo} from 'foo';
export {Foo as Bar};
```
`localNameNode` will be the `Bar` on the export line, and getting that symbol's declaration will point to `Foo` on the import line, which is what we want (so we can tell it's a re-export).

However, 
```ts
import {Foo} from 'foo';
export {Foo};
```
`localNameNode` will be the `Foo` on the export line, and getting that symbol's declaration will point right back at `Foo` on the _export_ line. Hence we can't get to the actual declaration of it on the import line. I don't know why this is. Using `getSymbolForName` (which searches symbols in scope at this location) lets us get to the declaration on the import line.